src/go/configgenerator/route_generator.go

```diff
@@ -336,15 +326,26 @@ func makeRouteTable(serviceInfo *configinfo.ServiceInfo) ([]*routepb.Route, []*r
 	return backendRoutes, methodNotAllowedRoutes, nil
 }
 
-func makeRoute(routeMatcher *routepb.RouteMatch, method *configinfo.MethodInfo, respTimeout time.Duration) *routepb.Route {
+func makeRoute(routeMatcher *routepb.RouteMatch, method *configinfo.MethodInfo) *routepb.Route {
```

Comment:
Up to line 399 is just moving some code into the function and cleaning up the function parameter. No behavior change.